### ISSUE TITLE
mobiscroll.widget.jqm.js cannot be used in combination jquery.mobile + requirejs

### ISSUE THREAD

**Issue by DzenisevichK** [acidb/mobiscroll#279]
`mobiscroll.widget.jqm.js` file contains:

```js
var ver = $.mobile && $.mobile.version && $.mobile.version.match(/1\.4/);
```

which depends on `$.mobile` which in case of (1.4.5 version) and `requirejs` will be `undefined` in release version (combined build) even when `shim` is specified:

```js
require.config({
	shim: {
		"external/mobiscroll": {
			deps: ["external/jquery", "external/jquery.mobile"]
		}
	}
});
```

**Comment by dioslaska** [comment]
Hello, here is an example, using RequireJS and jQuery Mobile. It works for me:

https://dl.dropboxusercontent.com/u/34664815/mobiscroll.require.jqm.zip

**Comment by DzenisevichK** [comment]
@dioslaska

Your example a bit not exactly which necessary.

Check this one version:
https://www.dropbox.com/s/fsnp7t9kyvtl0pt/mobiscroll.require.jqm.zip?dl=0

It built with `r.js` to reproduce the problem:
`node_modules\.bin\r.js.cmd -o build.js`
(command line for Windows)

I fixed `jquery.mobile` asynchronous loading + added `mobiscroll.mine.js` (not minified) and you can debug problem with it...

**Comment by dioslaska** [comment]
Fixed, will be released in 2.17.0

**Comment by DzenisevichK** [comment]
Ok, thank you!
